src/test/java/com/google/sps/CommentRetrievalTest.java

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import com.google.api.services.youtube.model.*;
+import com.google.sps.servlets.utils.YouTubeCommentRetriever;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class CommentRetrievalTest {
+
+  @Before
+  public void setup() {}
+
+  private final int HUNDRED = 100;
+  private final int EXCESS_HUNDRED = 200;
+
+  // A video that has sufficient comments for any test.
+  private final String POPULAR_VIDEO_URL = "dQw4w9WgXcQ";
+  // A video with very few comments
```

Comment:
Can you elaborate on how many comments? Like is sufficient >100, is few <20, etc